gatsby-node.js

```diff
@@ -4,18 +4,26 @@ const path = require('path');
 const fs = require('fs-extra');
 const { createFilePath } = require('gatsby-source-filesystem');
 const _ = require('lodash');
+// const { useTranslation } = require('react-i18next');
```

Comment:
What are the commented out additions in this file

Reply:
Some experimenting with i18n and plugins for `gatsby-node`.